doc/milestone.md

```diff
@@ -0,0 +1,21 @@
+# Milestone
+
+[M0] **Modelo del problema.**
+    -*Objetivo:* Analizar los HUs para extraer los diferentes elementos del dominio del problema y posteriormente codificarlo.
+    -*Entregable:* Codigo modularizado del problema.
+    -*Viabilidad:* Será viable si lo aprueba el product manager.
+
+[M1] **Implementación de una función que cree un horario para el alumno a partir de las asignaturas seleccionadas.**
+    -*Objetivo:* Implementar una clase o módulo que cree un horario para el alumno a partir de las asignaturas seleccionadas.
+    -*Entregable:* Codigo fuente de las clases o módulos.
+    -*Viabilidad:* Será viable si pasa los tests.
+    
+[M2] **Implementación de una función que a partir del horario que ha seleccionado el alumno, compruebe si hay incompatibilidades horarias entre las asignaturas que ha seleccionado.**
```

Comment:
Por favor, no pongáis "una función". Puede ser una o pueden ser 25. Referiros exactamente a lo que hay que hacer: implementar la lógica de negocio a partir del m0 que es el modelo del problema.